config.py

```diff
@@ -141,6 +30,22 @@ def __init__(self):
         if not os.path.exists(self.captionsFolder):
             os.makedirs(self.captionsFolder)
 
+        self.videoToUse = ''
+
+        self.saveFolder = "savedData"
+        self.fileTypes = ["topicModel", "topicsOverTime"]
```

Comment:
Optional: `"topicModel", "topicsOverTime"` make these as constants and import whenever needed

Reply:
So make it a dictionary like:
self.fileTypes = {'Topic Model':"topicModel", "Topics Over Time":"topicsOverTime"}
So that even if the file names are changed, nothing else in the code needs to be edited?

Reply:
This is what I meant: https://stackoverflow.com/questions/6343330/importing-a-long-list-of-constants-to-a-python-file. you could create `constants.py` where all the constants can be listed and exported if used in multiple files. If a constant used in a single file that declare at the top of the file. But this is optional.

Reply:
Ohhh. I like this methodology, but I suppose only if it's a lot of variables. I will make one if there are a lot of constants I define.